Makefile

```diff
@@ -104,23 +101,19 @@ lint-php-lint:
 
 lint-php-ncversion:
 	# Check min-version consistency
-	# TODO needs to be fixed
 	# php tests/nextcloud-version.php
 
 lint-php-phan:
 	# PHAN
-	# TODO needs to be fixed
-	# vendor/bin/phan --allow-polyfill-parser -k tests/phan-config.php --no-progress-bar -m checkstyle | vendor/bin/cs2pr --graceful-warnings --colorize
+	vendor/bin/phan --allow-polyfill-parser -k tests/phan-config.php --no-progress-bar -m checkstyle | vendor/bin/cs2pr --graceful-warnings --colorize
 
 lint-php-phpcs:
 	# PHP CodeSniffer
-	# TODO needs to be fixed
-	# vendor/bin/phpcs --standard=tests/phpcs.xml $(php_dirs) --report=checkstyle | vendor/bin/cs2pr --graceful-warnings --colorize
+	vendor/bin/phpcs --standard=tests/phpcs.xml $(php_dirs) --report=checkstyle | vendor/bin/cs2pr --graceful-warnings --colorize
```

Comment:
php-cs-fixer might actually be able to replace this with the composer scripts defined by running `composer run cs:check`